src/dipdup/datasources/evm_subsquid.py

```diff
@@ -92,50 +64,56 @@ async def iter_event_logs(
         first_level: int,
         last_level: int,
     ) -> AsyncIterator[tuple[SubsquidEventData, ...]]:
+        # FIXME: Earlier data has not indexed yet by Subsquid
+        first_level = max(first_level, 9_000_000)
```

Comment:
Better do this implicitly in config or just omit (nothing bad would happen except for slightly longer indexing, right?)